pkg/apis/scheduling/v1beta1/types.go

```diff
@@ -21,10 +21,8 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// PodGroupPhase is the phase of a pod group at the current time.
-type PodGroupPhase string
-
 // QueueState is state type of queue.
+// +kubebuilder:validation:Enum=Open;Closed;Closing;Unknown
```

Comment:
What if the QueueState is "",  Will there be an error?
IMO, the QueueState should not be validated because user can't set this filed and it set by controller internally, we just need validate fields that exposed to users.